test/test_helper.rb

```diff
@@ -39,20 +39,38 @@ def enrollment_file
   }
 end
 
-def statewide_file
-end
-
 def enrollment_statewide_files
   {
     enrollment: {
-      kindergarten: "./test/sample_kindergarten.csv"
+      kindergarten: "./test/sample_kindergarten.csv",
+      high_school_graduation: "./test/sample_high_school.csv"
     },
     statewide_testing: {
-      third_grade: "./test/sample_third_grade.csv"
+      third_grade: "./test/sample_third_grade.csv",
+      eighth_grade: "./test/sample_eighth_grade.csv",
+      math: "./test/sample_math.csv",
+      reading: "./test/sample_reading.csv",
+      writing: "./test/sample_writing.csv"
     }
   }
 end
 
+def headcount_analyst_files
+  {
+  enrollment: {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.